tests/test_meta.py

```diff
@@ -254,6 +254,38 @@ def test_commit_files(data_archive, cli_runner):
         assert r.exit_code == 44, r.stderr
 
 
+def test_commit_files_mine(data_archive, cli_runner):
+    with data_archive("points"):
```

Comment:
look into using `data_working_copy` for this test so it actually tests your new checkout code